pkg/cli/errors.go

```diff
@@ -15,7 +16,11 @@ func RunAndHandleError(ctx context.Context, cmd *cobra.Command) {
 	cmd.SilenceErrors = true
 	err := cmd.ExecuteContext(ctx)
 	if err != nil {
-		pterm.Error.Println(err)
+		errString := err.Error()
+		if cmd.Flag("project").Value.String() != "" {
+			errString = strings.Replace(errString, "namespace", "project", 1)
+		}
+		pterm.Error.Println(errString)
```

Comment:
Can we add a comment here explaining why we're doing this?

Reply:
yep added